package.json

```
@@ -52,7 +52,7 @@
   ],
   "peerDependencies": {
     "@babylonlabs-io/bbn-core-ui": "^0.8.0",
-    "@babylonlabs-io/btc-staking-ts": "0.4.0-canary.7",
+    "@babylonlabs-io/btc-staking-ts": "0.4.0-canary.9",
```

instead of bumping `btc-staking-ts` everytime you can use this:
```suggestion
    "@babylonlabs-io/btc-staking-ts": "0.4.x",
```

---

https://docs.npmjs.com/about-semantic-versioning